lib/oli/resources.ex

```diff
@@ -104,6 +104,13 @@ defmodule Oli.Resources do
 
   end
 
+  # returns a list of resource ids that refer to activity references in a page
```

Comment:
This context (Oli.Resources) does not seem to be the correct place for "activity aware" code